packages/application-shell/src/components/fetch-project/fetch-project.mc.graphql

```diff
@@ -13,6 +13,7 @@ query FetchProject($projectKey: String!) {
     }
     suspension {
       isActive
+      reason
```

Comment:
This was missing although we were using it here: https://github.com/commercetools/merchant-center-application-kit/blob/df68ce4bc5da71f9cbc763e356d777a6cbb4f6c6/packages/application-shell/src/components/project-container/project-container.js#L158